src/components/UserActions/UserActions.scss

```diff
@@ -44,6 +44,11 @@ $search-ui-dropdown-header-color: #1d4f76;
         margin: 0.5em 0;
     }
 
+    .coveo-no-actions {
```

Comment:
Euh? Out of scope I'd say ?

Reply:
Can we include this in the scope ?

Reply:
We can, sure, but can you update your pr description accordingly etc